CRM/Upgrade/Incremental/sql/4.7.beta1.mysql.tpl

```diff
@@ -14,7 +14,9 @@ INSERT INTO civicrm_navigation
 VALUES
 ( {$domainID}, 'civicrm/a/#/status', '{ts escape="sql" skip="true"}System Status{/ts}', 'System Status', 'administer CiviCRM', '', @parent_id, '1', NULL, 0 );
 
+-- CRM-17147 People with empty deceased-flag ('is null') get removed from recipient list of a mailing
 UPDATE civicrm_contact SET is_deceased = 0 WHERE is_deceased IS NULL;
+ALTER TABLE civicrm_contact ADD DEFAULT 0 FOR is_deceased;
```

Comment:
Actually the 1.1 ver written in xml (more like marked version when the field got introduced), is the point at which the field is added and doesn't mean that it will only be affected on 1.1. Because if you do a fresh installation you can see that the default is already set to 0 (as confirmed on my local). Such errorneous field only arise, if someone doesn't handle the upgrade logic in older version when the field was first introduced. So I think its ok to write upgrade logic for it if that's the case BUT the upgrade query is still failing. Check the related test build failure  https://test.civicrm.org/job/CiviCRM-Core-PR/6876/testReport/junit/(root)/CivicrmUpgradeTest/4_2_0_setupsh_sql_bz2/